### ISSUE TITLE
running example doesn't work

### ISSUE THREAD

**Issue by kitchWWW** [DigiScore/neoscore#59]
Steps to produce:

1) `mkdir test`
2) `cd test`
3) `python -m venv .`
4) `pip install neoscore`
5) `touch score.py`
6) pasting the following text into score.py:
```
from neoscore.common import *
neoscore.setup()
Text(ORIGIN, None, "Hello, neoscore!")
neoscore.show()
```
7) `python score.py` produces this error:
```
Traceback (most recent call last):
  File "score.py", line 1, in <module>
    from neoscore.common import *
  File "/Users/bse/Documents/cs/pythonws/neoscore/lib/python3.7/site-packages/neoscore/common.py", line 1, in <module>
    from neoscore.core import neoscore
  File "/Users/bse/Documents/cs/pythonws/neoscore/lib/python3.7/site-packages/neoscore/core/neoscore.py", line 17, in <module>
    from neoscore.core.mouse_event import MouseEvent
  File "/Users/bse/Documents/cs/pythonws/neoscore/lib/python3.7/site-packages/neoscore/core/mouse_event.py", line 38, in <module>
    @dataclass(frozen=True)
  File "/Users/bse/Documents/cs/pythonws/neoscore/lib/python3.7/site-packages/neoscore/core/mouse_event.py", line 56, in MouseEvent
    window_pos: tuple[int, int]
TypeError: 'type' object is not subscriptable
```

Other things as an FYI:
- `python --version` is `Python 3.7.5`
- `pip install PyQt5` returns `Requirement already satisfied`
- `pip install --upgrade pip` and `pip install --upgrade neoscore` both didn't solve the issue

**Comment by ajyoon** [comment]
Hey Brian, thanks for the report! It looks like there was a pasting mishap on the error you posted, could you provide the stacktrace? Also, what's your OS?

**Comment by kitchWWW** [comment]
macOS Monterey 12.3.1

**Comment by ajyoon** [comment]
Got it, I'll fix this shortly.

**Comment by ajyoon** [comment]
Okay, this is fixed in 0.1.5, which I just released. Try updating neoscore to latest and lmk if that did the trick!

**Comment by ajyoon** [comment]
closing on expectation this is resolved
